aiosmtpd/tests/test_smtp.py

```diff
@@ -1060,7 +1061,9 @@ def test_byclient(
         client.user = "goodlogin"
         client.password = PW
         auth_meth = getattr(client, "auth_" + mechanism)
-        if (mechanism, init_resp) == ("login", False):
+        if (mechanism, init_resp) == ("login", False) and (
+            sys.version_info < (3, 8, 9) or sys.version_info < (3, 9, 3)):
```

Comment:
The first comparison is a noop, given the second one. Maybe this should be:

```suggestion
            sys.version_info < (3, 8, 9) or (3, 9) < sys.version_info < (3, 9, 3)):
```

Reply:
Nice catch! I added `(3, 9) <=` to the second condition (though yours would work too since `(3, 9) < (3, 9, 0)` returns True). Seems correct now I think. 🐭️